### PR TITLE
New POST functionality added, new documentation added, functions commented and things explained

### PR DESCRIPTION
**Pull request by carlosmn1997**
The new functionality is calculating the MCM using a post request. It is described in description.md. His test is implemented too.

### CODE REVIEW

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 1]
```diff
@@ -12,10 +22,45 @@
...
+        
+            // http://ejerciciosresueltosprogramacion.blogspot.com/2017/01/funciones-minimo-comun-multiplo-de-dos.html
+            int m;
+            if (a > b)
+                m = a;
+            else
+                m = b;
+            while (m % a != 0 || m % b != 0)
+                m++;     
+
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: There is a more efficient way to do this. See https://cs.stackexchange.com/questions/1447/what-is-most-efficient-for-gcd

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 2]
```diff
@@ -12,10 +22,45 @@
...
+            // Return the result in the header
+            HttpHeaders responseHeaders = new HttpHeaders();
+            responseHeaders.add("result", Integer.toString(m));
+            return new ResponseEntity<>(responseHeaders, HttpStatus.OK);
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: This is not the right way to return a String. Think again.

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 3]
```diff
@@ -12,10 +22,45 @@
...
+        } catch(NumberFormatException e) { // Check if parameters aren't a number
+            HttpHeaders responseHeaders = new HttpHeaders();
+            return new ResponseEntity<>(responseHeaders, HttpStatus.BAD_REQUEST);
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: Can be simplified.

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 4]
```diff
@@ -12,10 +22,45 @@
...
     }
+
+    /**
+     * Function example to check the post functionality
+     * 
+     * @param form It must have two keys: "a":value1, "b":value2
+     */
+    @PostMapping("/mcm")
+    public ResponseEntity<String> mcm(@RequestParam MultiValueMap<String, String> form) {
+        try {
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: This can be helpful for you https://spring.io/guides/gs/handling-form-submission/

**Comment by carlosmn1997** [comment]
I have just sync my branch with the master branch.

**Pull request merged**